### PR TITLE
Session

### PR DESCRIPTION
**Pull request by makkhay**
completed everything

### CODE REVIEW

**Comment by makkhay** [comment]
finished everything.

File: src/members/PrakashGurung/Portfolio.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+      <div className="chart-container-prakash">
+
+        <div className="link-row">
+            <img className="link-icon" src={LinkIcon}></img>
+            <a className="link-title" href="https://google.com/" target="_blank">google.com</a>
+          </div>
+          <div className="link-row">
+            <img className="link-icon" src={ResumeIcon}></img>
+            <a href="https://google.com/" className="link-title" target="_blank">resume</a>
```
Comment (Monte9) [inline src/members/PrakashGurung/Portfolio.js]: link your resume here

File: src/members/PrakashGurung/Profile.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    src={GmailLogo}
+                    alt="Gmail Logo"
+                  />
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="AboutRightContainer">
+            <p className="IntroHello">About me</p>
+            <p className="AboutText"><span className="Bold AboutText Inline"></span> Young, independent and ambitious software engineer looking to make an impact.</p>
```
Comment (Monte9) [inline src/members/PrakashGurung/Profile.js]: please change this to your own text.

File: src/members/PrakashGurung/Profile.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            <p className="AboutText">-----</p>
+            <p className="AboutText">I like playing soccer. I'm in a soccer team and we play competative local soccer leagues and tournamnet. 
+            </p>
+            <p className="AboutText">
+              Besides soccer, I like playing guitar with my band. I listen and play all types of music but I prefer rock and heavy metal.
+            <p className="AboutText">
+            I'm really intersted in mobile app development. I learned android development on my own and I have published a couple of apps on playstore. 
+            </p>
+            <p className="AboutText">
+              Lastly, I really enjoy meeting new people and having a casual convo over
```
Comment (Monte9) [inline src/members/PrakashGurung/Profile.js]: please change this to your own text

**Comment by Monte9** [comment]
left a couple small text change comments. please address those comments when you get a chance.

**Pull request merged**